lib/shipit.rb

```diff
@@ -235,6 +236,13 @@ def task_logger
     @task_logger ||= Logger.new(nil)
   end
 
+  def respect_bare_shipit_file
+    @respect_bare_shipit_file = true if @respect_bare_shipit_file.nil?
+    !!@respect_bare_shipit_file
+  end
```

Comment:
Just use a regular accessor and then set the default with `self. respect_bare_shipit_file = true`